questdb_config_test.go

```
@@ -0,0 +1,222 @@
+	ctx := context.Background()
+
+	// Option wins over the connect-string key.
+	db, err := NewQuestDB(ctx, "ws::addr=127.0.0.1:1;sender_pool_max=2;lazy_connect=true;",
+		noopConnListener(), WithSenderPoolMax(7))
+	if err != nil {
+		t.Fatalf("build: %v", err)
+	}
+	if db.senderPool.maxSize != 7 {
+		t.Errorf("sender max=%d, want 7 (option over string)", db.senderPool.maxSize)
+	}
+	db.Close(ctx)
```

golangci-lint (2.12.2)

[error] Error return value of `db.Close` is not checked

(errcheck)